webapi/src/main/scala/org/knora/webapi/http/handler/KnoraExceptionHandler.scala

```diff
@@ -74,6 +75,21 @@ object KnoraExceptionHandler extends LazyLogging {
         }
       }
 
+    case bre: V2.BadRequestException =>
+      extractRequest { request =>
+        val url = request.uri.path.toString
+
+        if (url.startsWith("/v1")) {
+          complete(exceptionToJsonHttpResponseV1(bre, settingsImpl))
+        } else if (url.startsWith("/v2")) {
+          complete(exceptionToJsonHttpResponseV2(bre, settingsImpl))
+        } else if (url.startsWith("/admin")) {
+          complete(exceptionToJsonHttpResponseADM(bre, settingsImpl))
```

Comment:
This case could be removed because it would be equally handled in the `else` clause (or the `else` clause is wrong).

Reply:
agree